app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split(" ")) == 3:
+        command, file, new_file = command.split(" ")
+        if command == "cp":
+            if file != new_file:
+                with open(file, "r") as file_in, \
+                        open(new_file, "w") as file_out:
```

Comment:
Avoid using line breaking operator `\` it is a better practice to place conditions inside of `()`, e.g.:
```suggestion
                with (open(file, "r") as file_in,
                      open(new_file, "w") as file_out):
```
Apply this change to your code.